src/resources/packages/wizard/components/tabs.js

```diff
@@ -10,112 +10,216 @@ import * as TicketsContent from './tabs/tickets';
 import * as TecIcon from './icons/tec';
 
 const OnboardingTabs = ({ closeModal }) => {
+	/**
+	 * Initial collection of tabs.
+	 * @type {Array}
+	 *
+	 * @property {string}   name      - The name of the tab, used as the "slug".
+	 * @property {string}   title     - The title of the tab, displayed in the tab list.
+	 * @property {string}   className - The class name for the tab. This changes based on the tab's state.
+	 * @property {string}   baseClass - The base class name for the tab. This remains unchanged for reference.
+	 * @property {boolean}  disabled  - Whether the tab is disabled or not.
+	 * @property {function} content   - The content to render for the tab.
+	 */
 	const initialTabs = [
```

Comment:
Best practice for composing react components is to move static data like this (that don't change) outside of the component so it's not re-created _every-time_ the component re-renders.